### PR TITLE
Merge 5.1 BRANCH Admin Console Fixes

### PR DESCRIPTION
**Pull request by gerdogdu**
Mergin Admin console fixes from 5.1 BRANCH to master.

### CODE REVIEW

File: appserver/admingui/common/src/main/resources/appServer/jvmReport.jsf [hunk 1]
```diff
@@ -33,7 +33,10 @@
         setPageSessionAttribute(key="ep", value="/servers/server/#{pageSession.encodedInstanceName}/generate-jvm-report.json");
     />
     <sun:html id="html2"> 
-        <sun:head id="propertyhead" title="JVM Report" >
+        <sun:head id="propertyhead" title="JVM Report" javaScript="true" debug="false" parseOnLoad="false">
+            <h:outputScript name="jsf.js" library="javax.faces" target="head" />
```
Comment (smillidge) [inline appserver/admingui/common/src/main/resources/appServer/jvmReport.jsf:37]: Is the library javax.faces or jakarta.faces?

File: appserver/admingui/common/src/main/resources/logViewer/logViewer.jsf [hunk 2]
```diff
@@ -121,6 +121,7 @@
 </ui:event>
 <sun:html id="html2">
 <sun:head id="propertyhead" title="$resource{i18nc.logViewer.PageTitle}" debug="false" parseOnLoad="false">
+	<h:outputScript name="jsf.js" library="javax.faces" target="head" />
```
Comment (smillidge) [inline appserver/admingui/common/src/main/resources/logViewer/logViewer.jsf:124]: should the library be jakarta.faces?

File: appserver/admingui/common/src/main/resources/logViewer/logViewerRaw.jsf [hunk 3]
```diff
@@ -36,6 +36,7 @@
 </ui:event>
 <sun:html id="html2">
 <sun:head id="propertyhead" title="$resource{i18nc.logViewerRaw.PageTitle}" debug="false" parseOnLoad="false">
+	<h:outputScript name="jsf.js" library="javax.faces" target="head" />
```
Comment (smillidge) [inline appserver/admingui/common/src/main/resources/logViewer/logViewerRaw.jsf:39]: should the library be jakarta.faces?

Comment (gerdogdu) [reply appserver/admingui/common/src/main/resources/logViewer/logViewerRaw.jsf:39]: Yes, it must be jakarta.faces

Comment (gerdogdu) [reply appserver/admingui/common/src/main/resources/logViewer/logViewerRaw.jsf:39]: I will update these

Comment (gerdogdu) [reply appserver/admingui/common/src/main/resources/logViewer/logViewerRaw.jsf]: I have updated the files.

**Comment by gerdogdu** [comment]
I have updated the files as jakarta.faces

**Review by smillidge — APPROVED** [review]

**Pull request merged**